src/server/package.json

```diff
@@ -23,9 +23,15 @@
   "homepage": "https://github.com/michaelKurowski/lokim2#readme",
   "dependencies": {
     "body-parser": "^1.18.2",
+    "connect-mongo": "^2.0.1",
     "express": "^4.16.2",
+    "express-session": "^1.15.6",
+    "install": "^0.10.4",
     "mongoose": "^5.0.8",
     "mongoose-unique-validator": "^2.0.0",
+    "npm": "^5.7.1",
```

Comment:
npm?

Reply:
Probably I've updated npm to new version.